### ISSUE TITLE
`size` stops ignored in `style` definition defaults

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#568]
**TO REPRODUCE THE ISSUE, FOLLOW THESE STEPS:**

Make a new `style` and set a `draw` default including a `size` definition as stops:

```yaml
styles:
    my_points:
        base: points
        draw:
            color: red
            size: [[13, 14px], [16, 18px], [18, 19px]]
```

**RESULT:**

Points are not drawn unless the `size` is constant, eg `32px`. Stops are only respected under a `layer` declaration.

Example scene:

https://mapzen.com/tangram/play/?scene=https%3A%2F%2Fmapzen.com%2Fapi%2Fscenes%2F5%2F730%2Fresources%2Fblank.yaml#17.15007/37.79860/-122.41190

To test a fixed size, uncomment line 10: `size: 32px`
To test stops under a `layer`, uncomment line 19.

**Comment by nvkelso** [comment]
Aha! Good sleuthing :) /cc @sensescape

**Comment by nvkelso** [comment]
For background, we'd like to use the `size` stops to distribute standalone **icon library** ZIPs that include the draw defaults for Mapzen basemap cartography.

**Comment by bcamper** [comment]
Fixed in 4e101196b71396dc3825dd47fae13ef09763dcbe, will be in the next release.

**Comment by bcamper** [comment]
Released in [v0.13.0](https://github.com/tangrams/tangram/releases/tag/v0.13.0)
